lib/matplotlib/legend.py

```diff
@@ -348,7 +348,6 @@ def __init__(self, parent, handles, labels,
                  handler_map=None,
                  ):
         """
-
```

Comment:
This extra space was causing the hiccup in the formatting.  Sphinx is pretty fragile....